### PR TITLE
Replacing require.nil in web package

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Replacing require.nil in web package

#### Ticket Link
Not yet

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: web/webhook_test.go [hunk 1]
```diff
@@ -294,7 +294,7 @@ func TestCommandWebhooks(t *testing.T) {
 
 	for i := 0; i < 5; i++ {
 		response, appErr2 := http.Post(ApiClient.Url+"/hooks/commands/"+hook.Id, "application/json", bytes.NewBufferString(`{"text":"this is a test"}`))
```
Comment (wiggin77) [inline web/webhook_test.go:296]: This one shouldn't be called `appErr2` as it's an `error`.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:    One var rename requested.

File: web/webhook_test.go (hunk 1 above)
Comment (jespino) [reply web/webhook_test.go]: Done :)

**Review by agnivade — APPROVED** [review]

**Pull request merged**